### PR TITLE
update alerts displays for nTD and lfrTD

### PR DESCRIPTION
**Pull request by Yardantex**
name changes are more of a mechanical name, displaying boss emotes or
skill names for the alerts.

 - I had lots of merge conflicts all files should be checked carefully

As well fagging the translations that need to be checked or a missing in TODOs
French translations search for: transF
German translations search for : transG

### CODE REVIEW

File: Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Beligosh.lua [hunk 1]
```diff
@@ -107,14 +110,13 @@ BEL.Lang.Buff = {}
...
 BEL.Lang.Debuff = {}
 
-BEL.Lang.Debuff.VengeanceOfBeligosh = KBM.Language:Add("Vengeance of Beligosh")--tank debuff
-BEL.Lang.Debuff.VengeanceOfBeligosh:SetGerman("Rache von Beligosh")
-BEL.Lang.Debuff.VengeanceOfBeligosh:SetFrench("Vengeance de Beligosh")
-
-BEL.Lang.Debuff.SeedOfImmolation = KBM.Language:Add("Seed of Immolation")--raid debuff
-BEL.Lang.Debuff.SeedOfImmolation:SetGerman("Saat der Opferung")
-BEL.Lang.Debuff.SeedOfImmolation:SetFrench("Graine d'immolation")
+BEL.Lang.Debuff.Vengeance = KBM.Language:Add("Vengeance of Beligosh")--tank debuff
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Beligosh.lua]: Why has the variables names been changed? 
(VenganceOfBligosh and SeedOfImmolation)

File: Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Beligosh.lua [hunk 2]
```diff
@@ -148,7 +154,7 @@ function BEL:InitVars()
         CastBar = self.Beligosh.Settings.CastBar,
         EncTimer = KBM.Defaults.EncTimer(),
         PhaseMon = KBM.Defaults.PhaseMon(),
-        MechTimer = KBM.Defaults.MechTimer(),
+        -- MechTimer = KBM.Defaults.MechTimer(),
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Beligosh.lua]: Why has this been commented out?

File: Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Malannon.lua [hunk 3]
```diff
@@ -68,29 +68,34 @@ MAL.Lang.Ability.Meteor:SetGerman("Meteor")
...
 MAL.Lang.Verbose = {}
-MAL.Lang.Verbose.Meteor = KBM.Language:Add("Stack!")
-MAL.Lang.Verbose.Meteor:SetFrench("Pack!")
-MAL.Lang.Verbose.Meteor:SetGerman("Sammeln!")
+MAL.Lang.Verbose.Meteor = KBM.Language:Add("Meteor")
+-- TODO: transF, transG: Meteor (Check, got them from above)
+MAL.Lang.Verbose.Meteor:SetFrench("Météore")
+MAL.Lang.Verbose.Meteor:SetGerman("Meteor")
 
 MAL.Lang.Verbose.Blastback = KBM.Language:Add("Spread!")
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Malannon.lua]: Should this be changed as well?

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Beligosh.lua [hunk 4]
```diff
@@ -62,6 +61,9 @@ BEL.Beligosh = {
         },
         TimersRef = {
             Enabled = true,
+            SeedOfImmolation = KBM.Defaults.TimerObj.Create("purple"),
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Beligosh.lua]: This and next two lines seems to be duplicates.

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Beligosh.lua [hunk 5]
```diff
@@ -90,11 +92,16 @@ KBM.RegisterMod(BEL.ID, BEL)
 -- Ability Dictionary
 BEL.Lang.Ability = {}
 BEL.Lang.Ability.MagmaBreath = KBM.Language:Add("Magma Breath")
+<<<<<<< HEAD
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Beligosh.lua]: Leftover from Merging, needs to be removed.

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Beligosh.lua [hunk 6]
```diff
@@ -105,16 +112,12 @@ BEL.Lang.Verbose.Landing = KBM.Language:Add("Landing")
...
 
-
 BEL.Lang.Verbose.Enrage = KBM.Language:Add("Enrage")
 
-
 -- Buff Dictionary
 BEL.Lang.Buff = {}
 
 -- Debuff Dictionary
-BEL.Lang.Debuff = {}
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Beligosh.lua]: This shouldn't been removed

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Council.lua [hunk 7]
```diff
@@ -200,12 +199,13 @@ COF.Lang.Ability.NoRez = KBM.Language:Add("No Permission to Ressurect")
...
 -- Verbose Dictionary
 COF.Lang.Verbose = {}
-COF.Lang.Verbose.Flamescape = KBM.Language:Add("Go into the circle")
-COF.Lang.Verbose.Flamescape:SetGerman("In den weissen Kreis")
-COF.Lang.Verbose.Flamescape:SetFrench("Aller dans le cercle")
+COF.Lang.Verbose.Flamescape = KBM.Language:Add("Flamescape")
+-- TODO: transF, transG: Flamescape (check translation got it from above)
+COF.Lang.Verbose.Flamescape:SetGerman("Flammenformen")
+COF.Lang.Verbose.Flamescape:SetFrench("Pyroformation")
+>>>>>>> refs/remotes/kinnou02/master
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Council.lua]: Leftover from merge

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Council.lua [hunk 8]
```diff
@@ -323,6 +323,9 @@ end
 
 function COF:ResetTimers()
     KBM.MechTimer:AddRemove(self.Danazhal.TimersRef.Flamescape)
+    if not self.Danazhal.Dead then
+        KBM.MechTimer:AddStart(self.Danazhal.TimersRef.FirstFlamescape)
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Council.lua]: @kinnou02 should this be in here?

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Malannon.lua [hunk 9]
```diff
@@ -76,40 +76,53 @@ MAL.Lang.Ability.Meteor:SetFrench("Météore")
...
+-- TODO: transF: Supremacy (yellow)
 MAL.Lang.Verbose.MarkOfSupremacy:SetFrench("Allez dans le cercle jaune!")
+-- TODO: transG: Supremacy (yellow)
 MAL.Lang.Verbose.MarkOfSupremacy:SetGerman("Geh in den gelben Kreis!")
 
 MAL.Lang.Verbose.SiphonEnergy = KBM.Language:Add("Adds Spawn")
 
 MAL.Lang.Verbose.Runes = KBM.Language:Add("Runes of Power")
 
+>>>>>>> refs/remotes/kinnou02/master
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Malannon.lua]: Merge leftover

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua [hunk 10]
```diff
@@ -57,7 +57,9 @@ TAR.TarJulia = {
         AlertsRef = {
             Enabled = true,
             MoltenBlast = KBM.Defaults.AlertObj.Create("red"),
+            CallForHelp = KBM.Defaults.AlertObj.Create("blue"),
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua]: Believe this was supposed to be removed

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua [hunk 11]
```diff
@@ -57,7 +57,9 @@ TAR.TarJulia = {
...
             Fury = KBM.Defaults.AlertObj.Create("orange"),
+            SpiderWeave = KBM.Defaults.AlertObj.Create("purple"),
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua]: Believe this was supposed to be removed

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua [hunk 12]
```diff
@@ -301,12 +306,15 @@ function TAR:Start()
 
     KBM.Defaults.TimerObj.Assign(self.TarJulia)
 
-    self.TarJulia.MechRef.SpiderWeave = KBM.MechSpy:Add(self.Lang.Debuff.SpiderWeave[KBM.Lang], nil, "playerDebuff", self.TarJulia)
-    KBM.Defaults.MechObj.Assign(self.TarJulia)
-
     -- Create Alerts
     self.TarJulia.AlertsRef.MoltenBlast = KBM.Alert:Create(self.Lang.Verbose.MoltenBlast[KBM.Lang], 3, true, true, "red")
+    self.TarJulia.AlertsRef.CallForHelp = KBM.Alert:Create(self.Lang.Ability.CallForHelp[KBM.Lang], nil, true, true, "blue")
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua]: Believe this was supposed to be removed

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua [hunk 13]
```diff
@@ -301,12 +306,15 @@ function TAR:Start()
...
     self.TarJulia.AlertsRef.Fury = KBM.Alert:Create(self.Lang.Buff.Fury[KBM.Lang], 1, true, true, "orange")
+    self.TarJulia.AlertsRef.SpiderWeave = KBM.Alert:Create(self.Lang.Debuff.SpiderWeave[KBM.Lang], nil, true, true, "purple")
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua]: Believe this was supposed to be removed

File: Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Council.lua [hunk 14]
```diff
@@ -56,6 +56,9 @@ COF.Lang.Unit.Void = KBM.Language:Add("VOID")
 COF.Lang.Unit.Void:SetGerman("LEERE")
 COF.Lang.Unit.Void:SetFrench("NÉANT")
 
+-- Main Unit Dictionary
+COF.Lang.Unit = {}
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Council.lua]: This will cause issues

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Council.lua [hunk 15]
```diff
@@ -334,6 +337,7 @@ function COF:Death(UnitID)
     if self.Boldoch.UnitID == UnitID then
         self.Boldoch.Dead = true
         self.SetObjectives()
+        self:ResetTimers()
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Council.lua]: @kinnou02 ?

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua [hunk 16]
```diff
@@ -315,9 +323,11 @@ function TAR:Start()
     self.TarJulia.Triggers.MoltenBlast:AddTimer(self.TarJulia.TimersRef.MoltenBlast)
 
     self.TarJulia.Triggers.CallForHelp = KBM.Trigger:Create(self.Lang.Ability.CallForHelp[KBM.Lang], "cast", self.TarJulia)
+
+    self.TarJulia.Triggers.CallForHelp:AddAlert(self.TarJulia.AlertsRef.CallForHelp)
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua]: Believe this was supposed to be removed

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua [hunk 17]
```diff
@@ -315,9 +323,11 @@ function TAR:Start()
...
     self.TarJulia.Triggers.CallForHelp:AddTimer(self.TarJulia.TimersRef.CallForHelp)
 
-    self.TarJulia.Triggers.SpiderWeave = KBM.Trigger:Create(self.Lang.Debuff.SpiderWeave[KBM.Lang], "playerDebuff", self.TarJulia)
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua]: Why was this changed?

File: Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Beligosh.lua (hunk 1 above)
Comment (kinnou02) [inline Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Beligosh.lua]: it's probably a artefact from the merge, I originally named them like this before putting the long name.
It won't work as it is.

File: Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Council.lua [hunk 18]
```diff
@@ -191,9 +194,10 @@ COF.Lang.Ability.Void = KBM.Language:Add("Void")
 
 -- Verbose Dictionary
 COF.Lang.Verbose = {}
-COF.Lang.Verbose.Flamescape = KBM.Language:Add("Go into the circle")
-COF.Lang.Verbose.Flamescape:SetGerman("In den weissen Kreis")
-COF.Lang.Verbose.Flamescape:SetFrench("Aller dans le cercle")
+COF.Lang.Verbose.Flamescape = KBM.Language:Add("Flamescape")
```
Comment (kinnou02) [inline Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Council.lua]: We no longer need the `Verbose` version, we can directly use the ability name from `COF.Lang.Ability`

File: Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Malannon.lua [hunk 19]
```diff
@@ -68,29 +68,34 @@ MAL.Lang.Ability.Meteor:SetGerman("Meteor")
 
 -- Verbose Dictionary
 MAL.Lang.Verbose = {}
-MAL.Lang.Verbose.Meteor = KBM.Language:Add("Stack!")
-MAL.Lang.Verbose.Meteor:SetFrench("Pack!")
-MAL.Lang.Verbose.Meteor:SetGerman("Sammeln!")
+MAL.Lang.Verbose.Meteor = KBM.Language:Add("Meteor")
```
Comment (kinnou02) [inline Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Malannon.lua]: Same thing here, we can directly use the Ability name

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Council.lua (hunk 8 above)
Comment (kinnou02) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Council.lua]: This should be removed

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Council.lua (hunk 15 above)
Comment (kinnou02) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/Council.lua]: it isn't required but won't cause issue

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua [hunk 20]
```diff
@@ -301,12 +306,15 @@ function TAR:Start()
...
 
-    self.TarJulia.MechRef.SpiderWeave = KBM.MechSpy:Add(self.Lang.Debuff.SpiderWeave[KBM.Lang], nil, "playerDebuff", self.TarJulia)
-    KBM.Defaults.MechObj.Assign(self.TarJulia)
-
     -- Create Alerts
     self.TarJulia.AlertsRef.MoltenBlast = KBM.Alert:Create(self.Lang.Verbose.MoltenBlast[KBM.Lang], 3, true, true, "red")
+    self.TarJulia.AlertsRef.CallForHelp = KBM.Alert:Create(self.Lang.Ability.CallForHelp[KBM.Lang], nil, true, true, "blue")
     self.TarJulia.AlertsRef.Fury = KBM.Alert:Create(self.Lang.Buff.Fury[KBM.Lang], 1, true, true, "orange")
+    self.TarJulia.AlertsRef.SpiderWeave = KBM.Alert:Create(self.Lang.Debuff.SpiderWeave[KBM.Lang], nil, true, true, "purple")
+    self.TarJulia.MechRef.SpiderWeave = KBM.MechSpy:Add(self.Lang.Debuff.SpiderWeave[KBM.Lang], nil, "playerDebuff", self.TarJulia)
```
Comment (kinnou02) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua]: I prefer to have thing in that order: 
  - timers
  - mechspy
  - alerts
  - triggers

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua [hunk 21]
```diff
@@ -315,6 +318,7 @@ function TAR:Start()
     self.TarJulia.Triggers.MoltenBlast:AddTimer(self.TarJulia.TimersRef.MoltenBlast)
 
     self.TarJulia.Triggers.CallForHelp = KBM.Trigger:Create(self.Lang.Ability.CallForHelp[KBM.Lang], "cast", self.TarJulia)
+    self.TarJulia.Triggers.CallForHelp:AddAlert(self.TarJulia.AlertsRef.CallForHelp)
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua]: Shouldn't be here

File: Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua [hunk 22]
```diff
@@ -300,13 +302,14 @@ function TAR:Start()
     self.TarJulia.TimersRef.Fury = KBM.MechTimer:Add(self.Lang.Buff.Fury[KBM.Lang], 25)
 
     KBM.Defaults.TimerObj.Assign(self.TarJulia)
-
-    self.TarJulia.MechRef.SpiderWeave = KBM.MechSpy:Add(self.Lang.Debuff.SpiderWeave[KBM.Lang], nil, "playerDebuff", self.TarJulia)
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua]: This seems to be missing in PR

File: Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Beligosh.lua [hunk 23]
```diff
@@ -100,35 +102,37 @@ BEL.Lang.Verbose.BurningGround:SetFrench("Terrain enflammé")
...
 
 -- Debuff Dictionary
 BEL.Lang.Debuff = {}
 
-BEL.Lang.Debuff.VengeanceOfBeligosh = KBM.Language:Add("Vengeance of Beligosh")--tank debuff
-BEL.Lang.Debuff.VengeanceOfBeligosh:SetGerman("Rache von Beligosh")
-BEL.Lang.Debuff.VengeanceOfBeligosh:SetFrench("Vengeance de Beligosh")
+BEL.Lang.Debuff.VenganceOfBligosh = KBM.Language:Add("Vengeance of Beligosh")--tank debuff
+BEL.Lang.Debuff.VenganceOfBligosh:SetGerman("Rache von Beligosh")
+BEL.Lang.Debuff.VenganceOfBligosh:SetFrench("Vengeance de Beligosh")
```
Comment (Rathoz) [inline Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Beligosh.lua]: Variable name seems to be typo'd.
